typescript/core/src/v0/parse.ts

```diff
@@ -179,11 +177,12 @@ export function parseRecord({
       if (recordByteLength + reader.offset > recordView.byteLength) {
         throw new Error("Chunk records length exceeds remaining record size");
       }
+      // TODO: Something in at least the tests (if not elsewhere) is overwriting the
```

Comment:
NIT: I don't like leaving TODO in the code - would rather have this be an issue on github so we know where to follow up.